noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -482,16 +482,18 @@ contract Token {
             encode_and_encrypt_note_unconstrained(&mut context, user_keys.ovpk_m, user_keys.ivpk_m, user)
         );
         // 4. Now we get the note hiding points.
-        let mut fee_payer_point = TokenNote::hiding_point().new(
+        let fee_payer_partial_payload = TokenNote::partial_payload().new(
```

Comment:
I'm partial to destructuring, but feel free to ignore if you think this is clearer!

Reply:
I was not aware that there is a syntax for destructuring a struct. Is that what you have in mind?

Reply:
Oh shit, you're right it's a struct. I was thinking of a tuple...carry on

Reply:
I originally proposed it to be a tuple but it didn't work with macros.